src/Foundation/ArrayHelper.php

```diff
@@ -20,7 +20,7 @@ final class ArrayHelper
      *
      * @param mixed[] $arr
      */
-    public static function isAssoc(array $arr) : bool
```

Comment:
why these have changed? despite the fact I personally don't like it, I see no relevant changes in phpcs config. Most of the changed CS contains only this kind of changes

Reply:
`doctrine/coding-standard` was updated. And rules inside there changed to cause this. Do you want me to exclude that rule?